handler.cpp

```diff
@@ -121,9 +121,15 @@ RequestHandler::Status StaticHandler::HandleRequest(const Request& request, Resp
 
     // save content_type header based on requested file extension
     std::string content_type = get_content_type(abs_path);
-
     // raw byte array
     std::string to_send = read_file(abs_path);
+
+    // handling markdown
+    if (content_type == "markdown") {
+      markdown::Document d;
+      std::string s(d.read(to_send), {});
```

Comment:
where is s used? does this line of code mutate the state of the to_send string? I don't see string s being used to set the body of the response. Did you test that this renders the markdown to the browser as html?